grails-app/views/email/_approvalsAlert.gsp

```diff
@@ -0,0 +1,53 @@
+<%@ page contentType="text/html"%>
+<style>
+label {
+    font-weight: bold;
+}
+</style>
+<div>
+    <div class="header">
+        <g:render template="/email/header"/>
+    </div>
+    <div>
+        <g:message code='email.requestReceived.message' args='[requisition.destination, requisition.requestedBy, redirectUrl]' />
```

Comment:
nitpicky: let's use one type of `""` `''` - here you use a single one, and for others you use double one.